tests/hpc/ww4_await_pxe_install.pm

```diff
@@ -0,0 +1,48 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Compute node of warewulf4 cluster
+#     Once controller is up and running, run a VM which is provisioned
+#     by the controller, and run checks.
+# Maintainer: Kernel QE <kernel-qa@suse.de>
+
+use Mojo::Base qw(hpcbase), -signatures;
+use testapi;
+use lockapi;
+use utils;
+use mm_tests;
+use POSIX 'strftime';
+
+sub run ($self) {
+    my $i = 0;
+
+    while ($i < 10) {
+        check_screen('pxe-start');
+        record_info "pxe0", "pxe0";
+        sleep(1);
+        send_key "ctrl-B";
```

Comment:
Wouldn't be better to use `send_key_until_needlematch`? To avoid combination of `check_screen`, `sleep` and `send_key`.

Reply:
i can think of a proper way until now. `send_key_until_needlematch` doesnt seem to be the key tho. It couldnt handle the expected sequence of events. If `pxe-start` the send_key should be sent. Then wait a bit and reboot. The function, in the other side says, send some event and then match, which is of not help here. No? 
ofc i let it like that, but i am aware of the issue and i know that i should find a better way for this loop. but i dont have anything better at the moment and the implementation has taken longer than it was expected. Mainly i am saying, lets fix it in a follow up PR

Reply:
ok